source/quark/R/01_QuarkData.R

```diff
@@ -78,7 +78,10 @@ QuarkData <- setRefClass("QuarkData",
                              nImps        = "integer",
                              compFormat   = "character",
                              miDatasets   = "ANY",
-                             miceObject   = "ANY"
+                             miceObject   = "ANY",
+			     sameNaCntVec = "vector",
```

Comment:
Yes these three variables are used locally inside cleanCollinVars(). They will not persist after cleanCollinVars() but they need to be initialised locally . So shall I initilaize them locally inside the method cleanCollinVars()